pkg/ceph/rbd/connection.go

```diff
@@ -45,6 +45,7 @@ func NewConnection(monitor, id, key, pool, datapool string) (*Connection, error)
 	}, nil
 }
 
-func (c *Connection) Destroy() error {
-	return os.Remove(c.KeyFile)
+func Destroy() error {
```

Comment:
"Destroy" is scary, how about a more specific name?
```suggestion
func RemoveKeyDir() error {
```

Reply:
updated